.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# default (to be overridden by any matches in later lines below)
+*                        @rapidsai/jupyterlab-nvdashboard-write
+
+# CI code owners
+.flake8
```

Comment:
Should this line have a codeowner listed?

Reply:
ah, yes! This was a mistake. Just pushed a commit fixing that, thank you.